### PR TITLE
fixed bug in location command

### PR DESCRIPTION
**Pull request by Nystul-the-Magician**
location command no longer reports incorrectly that location index is invalid when it is correct and the player was teleported

### CODE REVIEW

**Comment by Interkarma** [comment]
Lypyl already has a nearly-complete fast travel map that will be integrated at a later date. Please reduce this pull request to just the fixes in location command.

File: Scripts/Game/DaggerfallUI.cs [hunk 1]
```diff
@@ -214,6 +218,13 @@ void ProcessMessages()
                     GameManager.Instance.PauseGame(true);
                     uiManager.PushWindow(dfCharacterSheet);
                     break;
+                case DaggerfallUIMessages.dfuiOpenAutomap:
+                    if (!GameObject.Find("Exterior")) // open automap only if player is in interior or dungeon - TODO: location automap for exterior locations
```
Comment (Interkarma) [inline Scripts/Game/DaggerfallUI.cs]: Should use PlayerEnterExit.IsPlayerInside here.

File: Resources/fallback.ini.txt [hunk 2]
```diff
@@ -36,8 +36,8 @@ StartInDungeon=False
 
 [Enhancements]
```
Comment (Interkarma) [inline Resources/fallback.ini.txt]: Please do not change fallback.ini.

**Comment by Interkarma** [comment]
Issues to resolve. See line comments.

**Comment by Nystul-the-Magician** [comment]
I never intended to add this to the pull request (how did I mess this up?) - I started automap (interior and dungeons) but I thought it is only in my own work in progress. the pull request was done before. somehow my commit and push afterwards seems to ended up in the previous pull request - but why?

**Pull request closed without merging**